:memo: fixing typos through the document and clarifying p1 labels (#2542)
kodadot/nft-gallery: CONTRIBUTING.md

KodaDot NFT gallery has a plan to be community-owned.
We are welcoming community contributions from you.
Take a sneak peek on [good first issues](https://github.com/kodadot/nft-gallery/contribute), comment, and make PR.
If everything goes well, chances that you will be **rewarded are high**. We might give retro-active bounties where the bounty label wasn't present if we like your commit.

**For better coordination, please join our [Development channel (#garage) on Discord](https://discord.gg/KkctBVQ8kP)**

## Getting started

Before you being:

> - As the development tool, we use **Node.js.** Check to see if you're on the version of **Node.js we support.**

Reviewer: ```suggestion
- We utilize **Node.js** as a development tool. To avoid potential compatibility issues, check if you're on the version of **Node.js we support.**
```

|                            50-100                            |                        150-300                        |                              450-600                              |                           700 - 1000                            |                                         1000+                                         |
| Beginner, few lines, <br /> lot of comments/assistance on PR | Required brain use, <br /> smart solution, autonomous | Complex, delivered new shiny stuff, <br /> did not break anything | Dedicated, new feature, rework older stuff, quick delivery time | You seems reliable, responsible, we can trust you, and you have alignment with project |

You can [check recent labels](https://github.com/kodadot/nft-gallery/labels)

## Hiring process

After you've **completed a few PRs, which we've merged to the upstream `main` branch**, you will be eligible to join our closer team.
We are happy to give you **one or two long-term tasks** to see how you can tackle more complicated and complex issues.

> The process behind this is to see if we **click together as part of the team in the long run.**

Reviewer: ```suggestion
The reasoning behind this is to see if we **click together as a team in the long run.**
```
